.github/workflows/build.yml

```diff
@@ -20,8 +20,16 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v3
 
-      - name: Setup cache
-        uses: Swatinem/rust-cache@v1
+      - name: Cache cargo registry
+        uses: actions/cache@v2
```

Comment:
Hi, have you seen? there's a v3 : https://github.com/actions/cache

Reply:
That's not this cache, it s this one: https://github.com/Swatinem/rust-cache which is specific to rust